CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+
+As a reminder, please take care to follow the Coding Standards for the project when adding any code-based contributions.
+
+Test your changes **thoroughly** before submitting a Pull Request. Whenever approrpriate, it is strongly recomended that you develop unit-tests with a good test spread to make sure that your new features are working as intended and are capable of handling anticipated edge cases. In cases where this is not possible, please be sure to follow a similar testing paradigm where both average and edge use cases are tested. In the long run, this will save the project a lot of tedious back-tracking and bug-smashing by proactively ensuring that easily-preventable bugs don't sneak into the master build.
+
+#### Build Configurations
+* Release - The release version of the app (live, user-facing build)
+* Debug - Debug version of the app (extra debug features enabled)
+
+### Submitting Changes
+
+#### Create a Pull Request
+Once you have finished building and testing your changes, it's time to submit them! From the /SplitScreen-Me/splitscreenme-nucleus repository, go to the Pull Requests section and click the "New Pull Request" button.
+
+**Make sure that the _base_ repository is set to "/SplitScreen-Me/splitscreenme-nucleus"!** 
+
+Otherwise, you will be attempting to merge your changes with a different repository. Double-check that you will be merging with the correct head branch as well (generally "master") Also verify that the _head_ repository is set to your fork and the branch of your fork where you made all the changes that you would like to implement.
+
+It is a good idea to create a detailed (but concise) comment to accompany the initial pull request to explain:
+* The issue the pull request is addressing
+* What is changed
+* Testing procedures used and their results
+* Potential issues or challenges encountered
+* Any questions or other comments for contribution reviewers
+
+#### Review Process
+All pull requests will approved by a sufficient number of project maintainers (those with write access to the project repository) before any changes are merged. Before aproving changes, reviewers will determine whether:
+* Changes do not interfere with existing features or design plans
+* There are overlap issues (i.e. other developers are already assigned to it) 
+* Sufficient testing occurred
+* Quality standards are upheld
+
+While it is certainly _possible_ that a quality pull request might be accepted right away, it should be expected that there will be some changes requested of the pull request, and issues that need to be discussed and resolved before it is approved. The pull request post itself is a great place to have these conversations, though contributors should not be afraid to directly message fellow developers either, when doing so is helpful to resolving any issues.

```

Comment:
```suggestion
While it is certainly _possible_ that a high quality pull request might be accepted right away,chagne requests, questions and discussions should be expected before the pull request can be merged. Discussions around the pull request should preferably be held in the pull request comments so it's easy for multiple people to follow the progress... but don't hesitate to reach out directly to the developers and maintainers either.
```

Reply:
Incorporating this one manually, since I have a couple proposed (very slight) tweaks - should be in the next commit of CONTRIBUTING.md.